src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { ReactComponent as Logo } from './logo.svg';
+
+const HeaderStyled = styled.header`
```

Comment:
Again I leave it to you, but what a lot of people do is to separate the styled components and the function component. Here you could have a Header.js file and a Header.style.js file that looks like

```
export const HeaderStyles = styled.header...

export const Nav = styled.nav...
```